lib/html_renderer.js

```diff
@@ -1,51 +1,30 @@
 let { HtmlRenderer } = require("commonmark");
-let Prism = require("prismjs");
-let loadLanguages = require("prismjs/components/");
-let preloadedLanguages = ["html", "css", "javascript"];
+let highlightSnippet = require("./highlight_snippet");
 
 class ImprovedHtmlRenderer extends HtmlRenderer {
-	constructor(...args) {
-		super(...args);
+	render(...args) {
 		this.examples = [];
```

Comment:
aside: this continues to trip me up; would "snippets" perhaps be a suitable alternative?

Reply:
Code Blocks would be the right terminology from the Markdown perspective.

Reply:
That seems even more confusing IMO; we're (intentionally) operating at a different level of abstraction here, so resorting to Markdown terminology would be conflating layers IMO.

Reply:
Renamed examples to snippets

Reply:
\o/ (sorry to be a pain)